lib/slax_web/controllers/blocker_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule SlaxWeb.BlockerController do
```

Comment:
Should this be removed or is it supposed to stick around?

Reply:
I kept it intentionally as I was thinking it would have real features eventually